spec/features/singup_spec.rb

```diff
@@ -0,0 +1,61 @@
+feature "Sign up" do
+  scenario "user can create an account" do
+    visit "/"
+    click_link "Sign up"
+    expect(page).to have_content "Sign Up"
+  end
+  scenario "user can see a Login link" do
+    visit "/signup"
+    find_link("Login").visible?
```

Comment:
The testing is great! I imagine you have 100% coverage? 
I can easily follow all the the descriptions as they are very clear.